app/src/main/java/com/trianglz/weatherapp/domain/usecases/countrysearch/ICountrySearchUseCase.kt

```diff
@@ -0,0 +1,9 @@
+package com.trianglz.weatherapp.domain.usecases.countrysearch
+
+import com.trianglz.weatherapp.domain.models.country.Country
+
+interface ICountrySearchUseCase {
```

Comment:
Please clarify the need to have an interface for a use case.

Reply:
to make testing the view model easier